policyengine_us/variables/gov/states/al/tax/income/exemptions/retirement/al_retirement_exemption_person.py

```diff
@@ -0,0 +1,21 @@
+from policyengine_us.model_api import *
+
+
+class al_retirement_exemption_person(Variable):
+    value_type = float
+    entity = Person
+    label = "Alabama retirement exemption for each person"
+    unit = USD
+    # The Code of Alabama Section 40-18-19 (a)(13).
+    documentation = "https://casetext.com/statute/code-of-alabama/title-40-revenue-and-taxation/chapter-18-income-taxes/article-1-general-provisions/section-40-18-19-exemptions-generally"
```

Comment:
add the form for this exemption, since the legal code's "taxable retirement income" is ambiguous